server/src/router.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  res.status(200).json({ message: "YO" });
+});
+
+if (process.env.NODE_ENV === "production") {
+  router.get("*", (req, res) => {
+    res.sendFile(__dirname + "/client/build/index.html");
```

Comment:
The whole program will need to be started from the root of the repo folder, right, for this to work? (Doesn't matter as such, just confirming)

Reply:
Thanks, good catch. I used router in `/api`, so this would have been misleading.